app/Models/Seat.php

```diff
@@ -14,6 +14,30 @@ class Seat extends Model
 
     public function location()
     {
-        return $this->belongsTo('App\Models\Location', 'location_id');
+        return $this->belongsTo(Location::class);
+    }
+
+    public function users()
+    {
+        return $this->belongsToMany(User::class)
+            ->withPivot('id', 'registration_date', 'is_full_day');
+    }
+
+    public function checkUsedSeat($date, $shift)
+    {
+        switch ($shift) {
+            case config('site.shift.all'):
+                return $this->users()->wherePivot('registration_date', $date)
+                    ->wherePivotIn('shift', [$shift, config('site.shift.morning'), config('site.shift.afternoon'),])
+                    ->exists();
+            case config('site.shift.morning'):
+                return $this->users()->wherePivot('registration_date', $date)
+                    ->wherePivotIn('shift', [config('site.shift.all'), $shift,])
+                    ->exists();
+            case config('site.shift.afternoon'):
+                return $this->users()->wherePivot('registration_date', $date)
+                    ->wherePivotIn('shift', [config('site.shift.all'), $shift,])
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Seat.php</br>Comma not allowed after last value in single-line array declaration